### PR TITLE
Cross server OOC

### PR DESCRIPTION
**Pull request by alexkar598**
pog, dont get your hopes up yet

### CODE REVIEW

**Comment by ynot01** [comment]
with who?

File: code/datums/world_topic.dm [hunk 1]
```diff
@@ -96,6 +96,32 @@
...
+	//SENDING THE MESSAGES OUT
+	for(var/c in GLOB.clients)
+		var/client/C = c // God bless typeless for-loops
+		if( (C.prefs.chat_toggles & CHAT_OOC) && (sourceadmin || !(sourcekey in C.prefs.ignoring)) )
+			var/sentmsg // The message we're sending to this specific person
+			if(C.holder) // If they're an admin-ish
+				sentmsg = oocmsg_toadmins // Get the admin one
+			else
+				sentmsg = oocmsg
+			sentmsg = "<span class='prefix'>RELAY: [input["source"]]</span>" + sentmsg
```
Comment (alexkar598) [inline code/datums/world_topic.dm]:
```suggestion
			sentmsg = "<span class='prefix'>RELAY: [input["source"]]</span> " + sentmsg
```

Comment (AshCorr) [reply code/datums/world_topic.dm]: why are you writing reviews on your own code...

**Comment by AshCorr** [comment]
I feel like this could be very confusing to players and potentially quite spammy.

**Comment by optimumtact** [comment]
just embed discord in an iframe duh

File: yogstation/code/modules/webhook/webhook.dm [hunk 2]
```diff
@@ -2,7 +2,7 @@
 	if(!CONFIG_GET(string/webhook_address) || !CONFIG_GET(string/webhook_key))
 		return
 
-	var/url = "[CONFIG_GET(string/webhook_address)]?key=[CONFIG_GET(string/webhook_key)]&method=[method]&data=[json_encode(data)]"
+	var/url = "[CONFIG_GET(string/webhook_address)]?key=[CONFIG_GET(string/webhook_key)]&method=[method]&data=[json_encode(data)]&server=[CONFIG_GET(string/cross_comms_name)]"
```
Comment (alexkar598) [inline yogstation/code/modules/webhook/webhook.dm]:
```suggestion
	var/url = "[CONFIG_GET(string/webhook_address)]?key=[CONFIG_GET(string/webhook_key)]&method=[method]&data=[json_encode(data)]"
```

File: code/modules/client/verbs/ooc.dm [hunk 3]
```diff
@@ -140,6 +140,16 @@ GLOBAL_VAR_INIT(mentor_ooc_colour, YOGS_MENTOR_OOC_COLOUR) // yogs - mentor ooc
...
 	//YOGS END
+	var/data = list()
+	data["normal"] = oocmsg
+	data["admin"] = oocmsg_toadmins
+	
+	var/source = list()
+	source["is_admin"] = !!holder
+	source["key"] = key
+	
+	//no need for source, we handle that ourself
```
Comment (alexkar598) [inline code/modules/client/verbs/ooc.dm]:
```suggestion
	
```

**Pull request merged**